openverse_catalog/dags/data_refresh/dag_factory.py

```diff
@@ -135,6 +135,22 @@ def _month_check(dag_id: str, session: SASession = None) -> str:
     )
 
 
+def _month_check_with_reporting(dag_id: str, media_type: str) -> str:
+    """
+    Wrapper for the monthly check function to report which step is starting
+    which step is next to slack.
```

Comment:
```suggestion
    Wrapper for the monthly check function to report which step is starting
    next to slack.
```

Reply:
Actually ignore this, @krysal's [suggestion](https://github.com/WordPress/openverse-catalog/pull/744#discussion_r984047545) is better! 😄